From a326ad006985626f94ea69630bf20dec03eb9ec7 Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Sun, 13 Jun 2021 01:17:17 +0200
Subject: [PATCH] Rename icon according to Flatpak convention

Flatpak prefers the app icon to be named after the app id, so itinerary
becomes org.kde.itinerary

Also move from 48 to scalable
---
 src/app/CMakeLists.txt                                          | 2 +-
 src/app/itinerary.notifyrc                                      | 2 +-
 src/app/main.cpp                                                | 2 +-
 src/app/org.kde.itinerary.desktop                               | 2 +-
 .../{48-apps-itinerary.svg => sc-apps-org.kde.itinerary.svg}    | 0
 5 files changed, 4 insertions(+), 4 deletions(-)
 rename src/app/{48-apps-itinerary.svg => sc-apps-org.kde.itinerary.svg} (100%)

diff --git a/src/app/CMakeLists.txt b/src/app/CMakeLists.txt
index b9df9c6..1132b64 100644
--- a/src/app/CMakeLists.txt
+++ b/src/app/CMakeLists.txt
@@ -177,7 +177,7 @@ else ()
 endif()
 
 install(TARGETS itinerary-app ${INSTALL_TARGETS_DEFAULT_ARGS})
-ecm_install_icons(ICONS 48-apps-itinerary.svg DESTINATION ${KDE_INSTALL_ICONDIR})
+ecm_install_icons(ICONS sc-apps-org.kde.itinerary.svg DESTINATION ${KDE_INSTALL_ICONDIR})
 if (NOT ANDROID)
     install(PROGRAMS org.kde.itinerary.desktop DESTINATION ${KDE_INSTALL_APPDIR})
     install(FILES org.kde.itinerary.appdata.xml DESTINATION ${KDE_INSTALL_METAINFODIR})
diff --git a/src/app/itinerary.notifyrc b/src/app/itinerary.notifyrc
index f9e8419..5ec60f4 100644
--- a/src/app/itinerary.notifyrc
+++ b/src/app/itinerary.notifyrc
@@ -1,5 +1,5 @@
 [Global]
-IconName=itinerary
+IconName=org.kde.itinerary
 DesktopEntry=org.kde.itinerary
 Name=KDE Itinerary
 Name[ca]=KDE Itinerary
diff --git a/src/app/main.cpp b/src/app/main.cpp
index 24a5359..209610c 100644
--- a/src/app/main.cpp
+++ b/src/app/main.cpp
@@ -206,7 +206,7 @@ int main(int argc, char **argv)
     }
 #endif
     QGuiApplication::setApplicationDisplayName(i18n("KDE Itinerary"));
-    QGuiApplication::setWindowIcon(QIcon::fromTheme(QStringLiteral("itinerary")));
+    QGuiApplication::setWindowIcon(QIcon::fromTheme(QStringLiteral("org.kde.itinerary")));
 
     QCommandLineParser parser;
     parser.addHelpOption();
diff --git a/src/app/org.kde.itinerary.desktop b/src/app/org.kde.itinerary.desktop
index fb352d3..672ad5b 100644
--- a/src/app/org.kde.itinerary.desktop
+++ b/src/app/org.kde.itinerary.desktop
@@ -48,7 +48,7 @@ GenericName[uk]=Керування маршрутом
 GenericName[x-test]=xxItinerary managementxx
 GenericName[zh_CN]=行程管理应用
 Exec=itinerary %f
-Icon=itinerary
+Icon=org.kde.itinerary
 Type=Application
 Comment=Travel assistance application.
 Comment[ca]=Aplicació per a l'assistència en viatge.
diff --git a/src/app/48-apps-itinerary.svg b/src/app/sc-apps-org.kde.itinerary.svg
similarity index 100%
rename from src/app/48-apps-itinerary.svg
rename to src/app/sc-apps-org.kde.itinerary.svg
-- 
2.32.0

